### PR TITLE
test: Make pybridge initial superuser behavior official

### PR DESCRIPTION
**Pull request by mvollmer**
The pybridge will only start a privileged bridge during startup when explicitly instructed in the "init" message.  This is now the officially correct behavior, although it differs from what the C bridge did.

Replaces #18984

### CODE REVIEW

File: test/verify/check-superuser [hunk 1]
```diff
@@ -438,13 +437,26 @@ class TestSuperuserOldWebserver(testlib.MachineCase):
...
+
+        else:
+            # The Python bridge will not start any privileged bridge
+            # if not explicitly told, and a old webserver (or old
+            # bridge) does not tell it anything.
+            b.check_superuser_indicator("Limited access")
+            b.go("/playground/test")
+            b.enter_page("/playground/test")
+            b.click(".super-channel button")
+            b.wait_in_text(".super-channel span", 'access-denied')
```
Comment (martinpitt) [inline test/verify/check-superuser:459]: Can you please add a check here that clicking on the priv indicator can actually get you root? I think that's the most important part of it.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! Can you please add the last bit of the test, and add "Fixes #18712" as well?

File: test/verify/check-superuser (hunk 1 above)
Comment (mvollmer) [reply test/verify/check-superuser:459]: Yes, good point.

**Comment by allisonkarlitskaya** [comment]
I think we need to talk about this a bit more.  Let's have a meeting, maybe?

**Comment by martinpitt** [comment]
[Lots of ruff errors](https://github.com/cockpit-project/cockpit/actions/runs/5421734041/jobs/9857546478?pr=19042#step:4:486) about broken f-strings

File: test/common/testlib.py [hunk 2]
```diff
@@ -867,17 +867,15 @@ def become_superuser(self, user: Optional[str] = None, password: Optional[str] =
 
         self.open_superuser_dialog()
 
-        pf_prefix = "" if self.machine.system_before(293) else "-v5"
-
         if passwordless:
-            self.wait_in_text(f".pf{pf_prefix}-c-modal-box:contains('Administrative access')", "You now have administrative access.")
-            self.click(f".pf{pf_prefix}-c-modal-box button:contains('Close')")
-            self.wait_not_present(f".pf{pf_prefix}-c-modal-box:contains('You now have administrative access.')")
+            self.wait_in_text(f"div[role=dialog]:contains('Administrative access')", "You now have administrative access.")
```
Comment (martinpitt) [inline test/common/testlib.py:871]: Nice solution!

**Review by martinpitt — COMMENTED** [review]
Aside from the f-string complaint this looks good to me, thank you!

**Comment by mvollmer** [comment]
> [Lots of ruff errors](https://github.com/cockpit-project/cockpit/actions/runs/5421734041/jobs/9857546478?pr=19042#step:4:486) about broken f-strings

Upps, fixed.

**Review by martinpitt — APPROVED** [review]
Danke!

**Review by allisonkarlitskaya — APPROVED** [review]
I am happy, thanks :)

**Comment by martinpitt** [comment]
Thanks! Let's land this -- even if we do change the behaviour in the future, that just means that we then can/will have to adjust these tests accordingly, which is a good thing.

**Pull request merged**